public_html/wp-content/themes/pattern-directory/src/components/patterns/index.js

```diff
@@ -16,6 +16,11 @@ import { store as patternStore } from '../../store';
 const Patterns = () => {
 	const query = useSelect( ( select ) => select( patternStore ).getCurrentQuery() );
 
+	if ( query ) {
+		query.locale = wporgLocale;
+		// todo why does it still say "loading all patterns"?
```

Comment:
This is happening because it's stuck in an infinite loop.

Move this code into the `QueryMonitor` control. Probably [here](https://github.com/WordPress/pattern-directory/blob/4d05a3f95e556ff4c64db43a9e920e3c544e558a/public_html/wp-content/themes/pattern-directory/src/components/query-monitor/index.js#L60).

Reply:
Thanks!